### PR TITLE
Added variety to the succeeding and failing authorizations.

### PR DESCRIPTION
**Pull request by AntonCroner**

### CODE REVIEW

File: authorization.spec.ts [hunk 1]
```diff
@@ -18,23 +18,29 @@ pax2payClient && (pax2payClient.organization = "agpiPo0v")
 describe("pax2pay.Authorization", () => {
 	let authorization: pax2pay.Authorization | undefined
 	let failedAuthorization: pax2pay.Authorization | undefined
+	const now = isoly.DateTime.now()
 	beforeAll(async () => {
-		const yesterday = isoly.DateTime.previous(isoly.DateTime.now(), { hours: 1 })
+		const yesterday = isoly.DateTime.previous(now, { hours: 1 })
```
Comment (marcushanses) [inline authorization.spec.ts:23]: Not really yesterday.

File: authorization.spec.ts [hunk 2]
```diff
@@ -45,39 +51,153 @@ describe("pax2pay.Authorization", () => {
...
+			city: "upcheck town",
+			zip: "12345",
+			address: "Streetname 1, 12345 Towncity",
+		},
+		acquirer: {
+			id: "2345erty",
+			number: "1351858913568",
+			country: "GB",
+		},
+		description: "An upcheck test authorization, to fail",
```
Comment (marcushanses) [inline authorization.spec.ts:201]: Could we write why they will fail in the description? It's hard to know which one should trigger what.

**Review by marcushanses — CHANGES_REQUESTED** [review]

File: authorization.spec.ts (hunk 1 above)
Comment (AntonCroner) [reply authorization.spec.ts:23]: yesterday is your fault, for that im making it yesterhour

File: authorization.spec.ts (hunk 2 above)
Comment (AntonCroner) [reply authorization.spec.ts]: true!

File: authorization.spec.ts (hunk 1 above)
Comment (marcushanses) [reply authorization.spec.ts]: omg.

**Review by marcushanses — APPROVED** [review]

**Pull request merged**